docs/en/faq.md

```diff
@@ -122,7 +122,9 @@ Read [JuiceFS Internals](development/internals.md) and [Data Processing Flow](in
 
 ### How to copy a large number of small files into JuiceFS quickly?
 
-You could mount JuiceFS with [`--writeback` option](reference/command_reference.mdx#mount-data-cache-options), which will write the small files into local disks first, then upload them to object storage in background, this could speedup coping many small files into JuiceFS.
+Mount JuiceFS with the [`--writeback` option](reference/command_reference.mdx#mount-data-cache-options),
```

Comment:
I know this is manual text wrapping, but since there's no similar wrapping in this repository, maybe merge these 3 lines into 1?

Reply:
If that's how you roll, sure. I contribute heavily to Ceph docs; this is my first time here.

I see someone else pushed a commit doing so.  I would have done that myself; now I'm at a loss of how to proceed, as I've never had good luck with PRs with multiple committers.